File: applied/1_hello_programming_world/answer/high.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Random rand = new Random();
        switch(rand.nextInt(3)) {
            case 0:
                System.out.printf("Hello\n");
                break;
            case 1:
                System.out.printf("World\n");
                break;
            case 2:
                System.out.printf("Again\n");
                break;
        }
    }
}

File: applied/1_hello_programming_world/answer/high.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main(void){
    srand((unsigned int)time(NULL));
    switch(rand() % 3) {
        case 0:
            printf("Hello\n");
            break;
        case 1:
            printf("World\n");
            break;
        case 2:
            printf("Again\n");
            break;
    }
}
